### ISSUE TITLE
Fresh install produces error on import

### ISSUE THREAD

**Issue by Lunrtick** [team-innovation/vue-sfc-rollup#12]
Hi

Building a package from a fresh 'lib' install leads to errors - the same errors that I've run into building my own package.

Basically, I run `sfc-rollup-init`
select 'lib'
cd, run 'yarn install'
change the package.json to use yarn instead of npm
run `yarn build`
See these errors for each different module type
```
(!) Unresolved dependencies
https://rollupjs.org/guide/en#warning-treating-module-as-external-dependency
vue-runtime-helpers/normalize-component.js (imported by src/lib-components/bulma-accordion-sample.vue)
vue-runtime-helpers/inject-style/browser.js (imported by src/lib-components/bulma-accordion-sample.vue)
(!) Missing global variable names
Use output.globals to specify browser global variable names corresponding to external modules
vue-runtime-helpers/normalize-component.js (guessing '__vue_normalize__')
vue-runtime-helpers/inject-style/browser.js (guessing '__vue_create_injector__')
created dist/bulma-accordion.umd.js in 98ms

```

and upon importing the built files, get errors stating that those dependencies are not found.

I feel like there's something very simple that I'm doing wrong, or that this is not expected behaviour.

I'm ready to provide any extra info if you need it

Cheers
Gabriel

**Comment by mgdodge** [comment]
Looks like rollup-plugin-vue published v4.4.0 a couple days ago, and that is breaking things. Seems that you've already [reported the issue there](https://github.com/vuejs/rollup-plugin-vue/issues/257), so you're probably also aware that until it is resolved, you can change the package.json dependency to remove the '^' from that, forcing exactly v4.3.2 to be installed.

Leaving this comment here for anyone else who comes here with the issue. In the meantime, I'll go ahead and lock in the versions here accordingly so others don't have the same issue.

**Comment by LeCoupa** [comment]
Updating `rollup-plugin-vue` to 4.6.0 fixed the issue for me.

**Comment by mgdodge** [comment]
Released v1.1.0 which updates rollup-plugin-vue to 4.6.2, now that things have stabilized. Closing this issue as it is resolved there.
